README.md

```diff
@@ -30,19 +30,17 @@
 
 ## Installation
 
-You can install this extension via:
+To install wasmtime-py, run this command in your terminal:
 
-```
-pip install wasmtime
+```bash
+$ pip install wasmtime
 ```
 
-Currently only x86\_64 Windows, macOS, and Linux are supported for this Python
-extension.
+The package currently supports x86\_64 Windows, macOS, and Linux.
```

Comment:
Would it make sense to also state the Python requirements here? I.e. "The package currently supports 64-bit builds of Python 3.6+ on x86\_64 Windows, macOS, and Linux"? (We do have Aarch64 support, too, but it seems like the install story there isn't yet fully sorted out.)

Reply:
(The 3.6+ is anticipating the merge of #21, of course)